### DISCUSSION TITLE
GrapesJS toolbar buttons are invisible

### DISCUSSION THREAD

**Discussion by kudryavtsevda** [GrapesJS/grapesjs#4172] [Q&A] [answered]
I am learning grapesjs throug getting started section. I attached the lates version of js and css 0.18.3. And I am not 100% successfull on the first step.  "Hello world component" doesn't show control buttons with remove, copy, etc buttons. What I did wrong?
![image](https://user-images.githubusercontent.com/4447809/156426524-b4bd4b59-3981-4df6-b8a8-29ff3550cbd2.png)

And I noticed that toolbar menus are commented due to some reason
![image](https://user-images.githubusercontent.com/4447809/156432067-7c67dd95-5d8b-40b0-9511-109a6c7c8bf3.png)

this is editor configuration
```
        const editor = grapesjs.init({
            container: '#@Html.IdFor(m => m.Html)-editor', // asp-net razor syntax
            fromElement: true,
            height: '300px',
            width: '90%',                               
            storageManager: false,
            panels: { defaults: [] },
        });

```

**Comment by artf** [comment, marked as answer]
Toolbar icons still rely on fontawesome icons (I'll try to replace them with their own icons in the next release), so I guess you have an issue with loading them (check your network tab in devtools)

**Reply by kudryavtsevda** [reply]
> Toolbar icons still rely on fontawesome icons (I'll try to replace them with their own icons in the next release), so I guess you have an issue with loading them (check your network tab in devtools)


Yes, it is connected with it and I have found out a workaround 
Thank you for your replay
